src/layout/FeaturesSection/FeaturesSection.svelte

```diff
@@ -66,6 +73,11 @@
 			description: $_("home.features.hashes.description", defaultI18nValues),
 			icon: Hash,
 		},
+		command: {
+			title: $_("home.features.command-palette.title", defaultI18nValues),
+			description: $_("home.features.command-palette.description", defaultI18nValues),
+			icon: Diamond,
```

Comment:
Should we use a better fitting icon here? The full list is here: https://icon-sets.iconify.design/

Reply:
I haven't found a better one but I'm happy to switch it if you have a recommendation.

Reply:
How about this? https://icon-sets.iconify.design/fluent/window-console-20-regular/

Reply:
I don't want it to be confused with a Terminal integration.

Reply:
https://icon-sets.iconify.design/fluent/color-24-regular/

Reply:
That looks like something we'd use for custom themes, ideally we'd create a custom icon but I'm not sure if we can use svg files in the current implementation.

Reply:
how about this? https://icon-sets.iconify.design/fluent/key-command-24-regular/
 i know it's a Mac key but it's recognizable

Reply:
That is an Apple-specific symbol, so I don't think it would fit in with Windows-specific applications.

Reply:
Let's move forward with the diamond icon for now. We can update it later if someone comes up with a more fitting icon.